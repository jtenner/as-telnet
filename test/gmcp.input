# test GMCP

# do GMCP
%FF%FD%C9

# send GMCP subnegotiation
%FF%FA%C9
Core.Hello {}
%FF%F0