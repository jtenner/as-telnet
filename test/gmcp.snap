SEND: %FF%FC%C9
SB: %C9 Core.Hello {}
